src/Sulu/Bundle/AdminBundle/Navigation/NavigationItem.php

```diff
@@ -282,12 +288,28 @@ public function getDisabled()
     }
 
     /**
+     * @return null|string
+     */
+    public function getDataNavigationUrl()
+    {
+        return $this->dataNavigationUrl;
+    }
+
+    /**
+     * @param string $dataNavigationUrl
+     */
+    public function setDataNavigationUrl($dataNavigationUrl)
+    {
+        $this->dataNavigationUrl = $dataNavigationUrl;
+    }
+
+    /**
      * Returns a copy of this navigation item without its children
      * @return NavigationItem
      */
     public function copyChildless()
     {
-        $new = new NavigationItem($this->getName());
+        $new = $this->newInstance();
```

Comment:
As you already wrote in your comment it is possible to make clone a deep copy by implementing the magic method. But I am not sure anymore if this is what @wachterjohannes wants to do anyway. Probably the name is just a bit confusing.